docs/Reminders.md

```diff
@@ -0,0 +1,10 @@
+??? ListItemButton vs ListItemLink vs MenuItemLink... maybe its just the naming that needs fixing
+??? When common folder vs component folder?
+??? calling useScrollStte position may be more correct than scroll
```

Comment:
Not sure I get what this means. Is this a possible improvement to my scrolling logic, which is a bit messy?

Reply:
Oh no, I think it was actually just nomenclature. To me "scroll" is the action, but the variable is really used to track the position on the page.